CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.98.0-beta.3] - 2020-04-15
+
+## [2.98.0-beta.2] - 2020-04-15
+
+## [2.98.0-beta.1] - 2020-04-15
+
+## [2.98.0-beta.0] - 2020-04-14
+
+## [2.98.0-beta] - 2020-04-14
```

Comment:
Before releasing the official you could remove these. This can be done doing:
```
git rebase -i origin/master
```
Then it will show something like:
```
pick a12ad435 Release v2.98.0-beta.1
```
Just change from `pick` to `d`, which will delete the commit